### DISCUSSION TITLE
How to add New Member on team ?

### DISCUSSION THREAD

**Discussion by denzizzy966** [frappe/gameplan#276] [General]
Hi, i have setup the gameplane and erpnext v15 and turn the erpnext to production mode. but how to add new member on team or project ? because the add new member doesnt show up on member list gameplane

**Comment by stephenBDT** [comment]
I have the same problem

![image](https://github.com/user-attachments/assets/9a34868a-5bbd-46a7-b57e-de46cffbff02)


And my instance has a lot of Users

**Comment by stephenBDT** [comment]
# The solution is, to give all the users the GP Member Role!

![image](https://github.com/user-attachments/assets/56331316-ab57-4e65-bdcb-56d0168d77c9)
